src/models/query.js

```diff
@@ -0,0 +1,61 @@
+class Query {
+    /**
+     * [constructor description]
+     * @param  {String} name [description]
+     * @return {Query Class}      [description]
```

Comment:
you do not return a query class you should return a query instance. but you do not return anything :p 
Remove unused descriptions